ci-operator/step-registry/rosa/aws/sts/hypershift/cilium/rosa-aws-sts-hypershift-cilium-chain.yaml

```diff
@@ -0,0 +1,21 @@
+chain:
+  as: rosa-aws-sts-hypershift-cilium
+  env:
+  - name: HOSTED_CP
+    default: "true"
+  - name: NO_CNI
+    default: "true"
```

Comment:
I do advise not to add env in the chain for hard maintenance. Workflow and job-level are enough.

Reply:
This chain/workflow is named with cilium that means NO_CNI should be enabled by default, right?